docs/developers/listener.md

```diff
@@ -0,0 +1,181 @@
+
+---
+title: Working With Events
+---
+
+Listening to events with Velocity's `@Subscribe` annotation is straightforward. You've already seen one such listener, using the ProxyInitializeEvent in your main class.
+
+### Adding listening methods
+
+To listen to an event, mark the method with `@Subscribe`, like shown. This works similarly to annotation-driven event listening
+in other APIs you may be familiar with; it's the equivalent of Bukkit's/Bungee's @EventHandler and Sponge's @Listener.
+
+```java
+@Subscribe(
+public void onPlayerChat(PlayerChatEvent event) {
+	// do stuff
+}
+```
+
+Note well that the import is `com.velocitypowered.api.event.Subscribe` and *NOT* `com.google.common.eventbus`.
```

Comment:
Can we use the styling that I introduced for this?

Reply:
Yes. Does it work well with code blocks?

If so, I would replace this sentence with
```
<div>
<div class="caution-header">
    <img src="https://cdn.discordapp.com/attachments/734487433621668011/736277965142491276/warning_icon.png"></img>
     Caution
</div>
  <div class="caution">
      Note well that the import is `com.velocitypowered.api.event.Subscribe` and *NOT* `com.google.common.eventbus`.
  </div>
</div>
```

Further, do you intend to use the discord cdn's `warning_icon.png` or the repository's `warning.png`?

Reply:
it will work like that as long as you follow how i made mine. Also, i have that in there temporarily (it's just hosted from the Anvil discord in a channel that we use for hosting images)